Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,57 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  url "https://github.com/MariaDB/mariadb-connector-odbc/archive/2.0.13.tar.gz"
```

Comment:
That's still marked as Alpha, it might be better to wait until upstream considers it stable ?

It would also require updating the mariadb-connector-c formula to 3 or higher, which is currently at 2.2.2. The current upstream stable version for that is at 2.3.2, which is still not sufficient to compile the 3.0 odbc connector.

So you'd be upgrading at least two packages to an alpha version.